flink-connector/README.md

```diff
@@ -54,15 +66,213 @@ their pom.xml file by adding:
 <dependency>
   <groupId>com.google.pubsub.flink</groupId>
   <artifactId>flink-connector-gcp-pubsub</artifactId>
-  <version>0.0.0</version>
+  <version>1.0.0-SNAPSHOT</version>
 </dependency>
 ```
 
 Learn more about Flink connector packaging
 [here](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/configuration/overview/).
 
-## Documentation
+## Quickstart
+
+This section describes how to build the example Flink job located under
+`flink-examples-gcp-pubsub/`, deploy it to a Flink cluster, and verify that it
+is running correctly. The example job streams data from a Pub/Sub subscription
+source and publishes that data to a Pub/Sub topic sink.
+
+### Prerequisites
+
+1.  Flink cluster
+
+    Flink's
+    [first steps guide](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/try-flink/local_installation/)
+    describes how to download Flink and start a local-running Flink cluster to
+    which you can deploy the example Flink job.
+
+2.  [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+    in which you can create
+    [Pub/Sub topics](https://cloud.google.com/pubsub/docs/create-topic) and
+    [Pub/Sub subscriptions](https://cloud.google.com/pubsub/docs/create-subscription)
+
+3.  [OPTIONAL] `gcloud` CLI
+    ([installation instructions](https://cloud.google.com/sdk/docs/install))
+
+    For your convenience, this section includes `gcloud` commands that you can
+    run to complete certain steps. These steps can also be completed using the
+    [GCP console UI](https://cloud.google.com/cloud-console) or using one of the
+    [Pub/Sub client libraries](https://cloud.google.com/pubsub/docs/reference/libraries).
+
+### Building the Example
+
+Flink jobs are typically packaged into a JAR file with their dependencies. The
+following commands use Maven to build and package the example Flink job into a
+JAR file that will be deployed to run on a Flink cluster.
+
+```sh
+# If you haven't already done so, download this repository.
+git clone https://github.com/GoogleCloudPlatform/pubsub.git
+cd pubsub/flink-connector
```

Comment:
do users still have to build the whole flink-conenctor project to build the example jar?

Reply:
Yes. Until we publish the connector to a Maven repo, it needs to be built from source.